dev/com.ibm.ws.anno/publish/jandex/runJandex.bat

```diff
@@ -0,0 +1,89 @@
+setLocal
```

Comment:
Those were used to generate jandex indexes for unit testing jandex function.  We kept them to remember the steps we used to work with jandex.

Reply:
Ideally it would be added as a gradle task in a build.gradle and then you wouldn't have to have a batch file and shell file for this because you could just call the build.gradle to run the jandex code.  Are you planning on changing this as part of this PR?

Reply:
Being tracked for future cleanup in #9225.